MediaPlayer/MediaPlayer.js

```diff
@@ -261,37 +294,41 @@ const MediaPlayerBase = kind({
 		durFmt: ({locale}) => getDurFmt(locale)
 	},
 
-	render: ({currentTime, durFmt, loop, mediaComponent, mediaRef, onChange, onEnded, onNext, onPause, onPlay, onPrevious, onRepeat, onShuffle, onUpdate, paused, playlist, proportionPlayed, repeat, shuffle, sourceIndex, total, ...rest}) => {
+	render: ({currentTime, disabled, durFmt, loop, mediaComponent, mediaRef, onChange, onEnded, onNext, onPause, onPlay, onPrevious, onRepeat, onShuffle, onUpdate, paused, playlist, proportionPlayed, repeat, shuffle, sourceIndex, spotlightDisabled, total, ...rest}) => {
 		return (
 			<div {...rest}>
-				<Media
-					loop={loop}
-					mediaComponent={mediaComponent}
-					onEnded={onEnded}
-					onUpdate={onUpdate}
-					ref={mediaRef}
-					source={playlist ? playlist[sourceIndex] : null}
-				/>
-				<MediaSlider
-					onChange={onChange}
-					value={proportionPlayed}
-				/>
-				<Times
-					current={currentTime}
-					formatter={durFmt}
-					total={total}
-				/>
-				<MediaControls
-					onNext={onNext}
-					onPause={onPause}
-					onPlay={onPlay}
-					onPrevious={onPrevious}
-					onRepeat={onRepeat}
-					onShuffle={onShuffle}
-					paused={paused}
-					repeat={repeat}
-					shuffle={shuffle}
-				/>
+				<Container spotlightDisabled={spotlightDisabled}>
```

Comment:
Can we not add another div here? I guess we could do like,
<Container {...rest} spotlightDisabled={spotlightDisabled}>